types/recharts/index.d.ts

```diff
@@ -310,6 +310,7 @@ export interface PieProps extends Partial<CSSStyleDeclaration> {
 	minAngle?: number;
 	paddingAngle?: number;
 	nameKey?: string;
+	dataKey?: string;
```

Comment:
where exactly in the documentation is this property?

Reply:
You are right, it is not yet in the official documentation, but see:

Official demo: https://github.com/recharts/recharts/blob/master/demo/component/PieChart.js#L148